templates/showcase.html

```diff
@@ -23,12 +23,12 @@ <h2>Projects using Vanilla</h2>
   </div>
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <img src="https://assets.ubuntu.com/v1/25df3b87-ubuntu-vf.io-showcase.png" alt="">
```

Comment:
Repeating "Showcase screenshot" over and over again is not really helpful.
As each example has its own text it should be sufficient as description of given page. Images don't provide any additional value (other than visual), so are completely fine with empty alt text.